ci/report/report.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
```

Comment:
Should this code have a particle copyright notice?

Reply:
It's not ours. I made improvements to make it work with Travis CI, but the original work is borrowed from the `cpp-coveralls` project.

I put it in it's own folder, along with it's LICENSE agreement, APL2.0, to highlight this fact.

Reply:
ok, thanks for clarifying!